presto-docs/src/main/sphinx/release/release-339.rst

```diff
@@ -0,0 +1,77 @@
+=========================
+Release 339 (XX Jul 2020)
+=========================
+
+General Changes
+---------------
+
+* Add :func:`approx_most_frequent`. (:issue:`3425`)
+* Physical bytes scan limit for queries can be configured via ``query.max-scan-physical-bytes`` configuration property
+  and ``query_max_scan_physical_bytes`` session property. (:issue:`4075`)
+* Remove support for addition and subtraction between ``TIME`` and ``INTERVAL YEAR TO MONTH`` types. (:issue:`4308`)
+* Fix planning failure when join criteria contains subqueries. (:issue:`4380`)
+* Fix failure when subqueries appear in window function arguments. (:issue:`4127`)
+* Fix failure when subquery in ``WITH`` clause contains hidden columns. (:issue:`4423`)
+* Fix failure when referring to type names with different case ``GROUP BY`` queries. (:issue:`2960`)
```

Comment:
Something seems missing. Should this be something like

> Fix failure when referring to type names with differing case in a GROUP BY clause.

Reply:
Yeah, that was mean to say "in GROUP BY queries". Updated with your wording